src/analyzing/AST_to_IL.ml

```diff
@@ -112,6 +112,15 @@ let fixme_stmt kind gany =
 (*****************************************************************************)
 (* Helpers *)
 (*****************************************************************************)
+
+let treats_expr_as_stmt (lang : Lang.t) =
+  match lang with
+  | Rust
+  | Julia
+  | Ruby ->
```

Comment:
Maybe add OCaml here already?

Reply:
This code was removed entirely, so I'll just mark this resolved.